src/api/app/models/package.rb

```diff
@@ -21,6 +20,15 @@ class Package < ApplicationRecord
                        '.taz', '.tb2', '.tbz', '.tbz2', '.tgz', '.tlz', '.txz', '.ucode',
                        '.xpm', '.xz', '.z', '.zip', '.ttf'].freeze
 
+  ThinkingSphinx::Callbacks.append(self, behaviours: [:real_time]) do |package|
+    # Index package only if its name, title or description changed
+    if package.name_previously_changed? || package.title_previously_changed? || package.description_previously_changed?
+      [package]
```

Comment:
can you move those callbacks to a concern? like https://api.rubyonrails.org/classes/ActiveSupport/Concern.html ?

Reply:
Sure!